CloudKitty/api/Dockerfile

```diff
@@ -7,7 +7,12 @@ RUN  apt-get update \
         gcc \
         python3-dev \
     && pip install cloudkitty \
-    && mkdir /etc/cloudkitty \
+    && mkdir \
+        /etc/cloudkitty \
+        /etc/cloudkitty-config \
+    && ln -sf \
+        /etc/cloudkitty/cloudkitty.conf \
+        /etc/cloudkitty-config/cloudkitty.conf \
```

Comment:
We also need a symbolic link for `metrics.yml` then

Reply:
Good remark